pkg/suggestion/v1beta1/nas/darts/service.py

```diff
@@ -134,3 +140,60 @@ def get_algorithm_settings(settings_raw):
         algorithm_settings_default[s_name] = s_value
 
     return algorithm_settings_default
+
+
+def validate_algorithm_settings(spec: api_pb2.ExperimentSpec) -> (bool, str):
+    # Validate Operations
+    is_valid, message = validate_operations(spec.nas_config.operations.operation)
+    if not is_valid:
+        return False, message
+
+    # Validate AlgorithmSettings
+    is_valid, message = validate_algorithm_spec(spec.algorithm.algorithm_settings)
+    if not is_valid:
+        return False, message
+
+    return True, ""
+
+
+def validate_algorithm_spec(algorithm_settings: list[api_pb2.AlgorithmSetting]) -> (bool, str):
+    for s in algorithm_settings:
+        try:
+            if s.name == "num_epochs":
+                if not int(s.value) > 0:
+                    return False, "{} should be greate than zero".format(s.name)
+
+            # Validate learning rate
+            if s.name in ["w_lr", "w_lr_min", "alpha_lr"]:
+                if not float(s.value) >= 0.0:
+                    return False, "{} should be greate or equal than zero".format(s.name)
```

Comment:
```suggestion
                    return False, "{} should be greater than or equal to zero".format(s.name)
```

there are same typos in this file, so you may prefer to use your own IDE to replace this.